third_party/model_api/BUILD

```diff
@@ -0,0 +1,53 @@
+load("@rules_foreign_cc//foreign_cc:cmake.bzl", "cmake")
+
+_ALL_CONTENT = """\
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+"""
+
+
+
+visibility = ["//visibility:public"]
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["model_api/cpp/**"]),
+    visibility = ["//visibility:public"],
+)
+
+cmake(
+    name = "model_api_cmake",
+    cache_entries = {
+        "CMAKE_POSITION_INDEPENDENT_CODE": "ON",
+        "OpenVINO_DIR": "/opt/intel/openvino/runtime/cmake",
+    },
+    env = {
+        "HTTP_PROXY": "http://proxy-dmz.intel.com:911",
```

Comment:
don't include proxy

Reply:
I believe proxy is required by cmake to download dependencies.